model/config.go

```diff
@@ -720,8 +783,16 @@ func (s *SqlSettings) SetDefaults() {
 		s.DataSource = NewString(SQL_SETTINGS_DEFAULT_DATA_SOURCE)
 	}
 
-	if len(s.AtRestEncryptKey) == 0 {
-		s.AtRestEncryptKey = NewRandomString(32)
+	if s.DataSourceReplicas == nil {
+		s.DataSourceReplicas = []string{}
```

Comment:
Probably a naive question, but is there a reason we aren't making these list-type variables into pointers too?

Reply:
Makes perfect sense to me, thanks.